gen3/bin/dbbackup.sh

```diff
@@ -43,19 +36,37 @@ vpc_name="$(gen3 api environment)"
 namespace="$(gen3 db namespace)"
 sa_name="dbbackup-sa"
 bucket_name="gen3-db-backups-${account_id}"
+bucket_name_encrypted="gen3-db-backups-encrypted-${account_id}"
+kms_key_alias="alias/gen3-db-backups-kms-key"
+
+cluster_arn=$(kubectl config current-context)
+eks_cluster=$(echo "$cluster_arn" | awk -F'/' '{print $2}')
 
-gen3_log_info "policy_name: $policy_name"
 gen3_log_info "account_id: $account_id"
 gen3_log_info "vpc_name: $vpc_name"
 gen3_log_info "namespace: $namespace"
 gen3_log_info "sa_name: $sa_name"
 gen3_log_info "bucket_name: $bucket_name"
```

Comment:
We should not need a unencrypted bucket,  we should only allow an encrypted bucket.